views/siop.jade

```diff
@@ -0,0 +1,36 @@
+doctype html
+html
+    head
+        title #{title}
+        meta(name="viewport" content="width=device-width, initial-scale=1")
+    body
+        main
+            block header
+                header.header
+                    a.title(href='/')
+                        h1 #{title}
+
+            div.container-siop
+                div.content-row
+                    h2 Please authenticate using the QR in the popup.
+    script.
+                function openWindow() {
```

Comment:
We also need to provide a client_id. This is functionallity implemented just this week. See: https://github.com/FIWARE/VCVerifier/blob/main/api/api.yaml#L20
We will need a config-parameter to configure that, too.

Reply:
Ok, got it. I'll adjust it